S32-list/create.t

```diff
@@ -22,7 +22,17 @@ isa-ok($list_obj, List, 'Creating a new list object with new works.');
 is($list_obj, list(4, 5, 6), 'The list object contains the right values.');
 is(+$list_obj, 3, 'Finding the length functions properly.');
 }
-
+{
+my @list-pos is List = 1, 2, 3;
```

Comment:
I see the reason why you don't indent here, but to my view it's either of two would be better options:

- indent the old test code
- don't use blocks

Considering that you're referring to an existing PR, indenting the blocks is preferable if accompanied with a comment referring to the PR page. Something like:

```raku
# https://github.com/rakudo/rakudo/pull/4208
{
    ...; # test code
}
```

If the PR, in turn, fixes an existing ticket, then a link to the ticket would be a better choice.